### PR TITLE
Upgrade libp2p deps

### PR DESCRIPTION
**Pull request by hanabi1224**
**Summary of changes**
Changes introduced in this pull request:
- Upgrade libp2p deps version to latest

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

**Review by creativcoder — APPROVED** [review]

File: base/src/numbers.rs [hunk 1]
```diff
@@ -173,17 +173,17 @@ pub struct BlockTime(pub u64);
 
 impl BlockTime {
     /// Unix timestamp conversion (seconds since the unix epoch)
-    pub fn from_unix_epoch(ts: u64) -> Self {
+    pub const fn from_unix_epoch(ts: u64) -> Self {
```
Comment (willemolding) [inline base/src/numbers.rs:176]: Unrelated changes :P 

Cool to have anyway

**Review by willemolding — APPROVED** [review]
